DESCRIPTION

```diff
@@ -24,7 +24,7 @@ BugReports: https://github.com/GSK-Biostatistics/tfrmt/issues
 Depends: 
     R (>= 4.1.0)
 Imports: 
-    cards (>= 0.6.1.9010),
+    cards,
```

Comment:
this needs some sort of minimum version because the functions don't run with the older versions of cards